src/mole_C++/laplacian.h

```diff
@@ -1,19 +1,59 @@
+/**
+ * @file laplacian.h
+ * 
+ * @brief Mimetic Laplacian Constructors
```

Comment:
nit: Not just constructors, class definition is also here.

Reply:
Yep

https://github.com/csrc-sdsu/mole/blob/0295c8fdbeebed3a51260cb1276d185a8270844d/src/mole_C%2B%2B/laplacian.h#L1-L8